### ISSUE TITLE
How can I fine-tune the model?

### ISSUE THREAD

**Issue by LeBron-Jian** [Raessan/object_detection_dinov3#1]
I trained my own dataset; however, I found that the losses for cls and ctr decreased normally, while the loss for reg remained at around 0.99 and did not decrease. How did you fine-tune it?

<img width="1672" height="318" alt="Image" src="https://github.com/user-attachments/assets/1d79eb2f-36d3-4a7d-8336-050937105ccd" />
